lib/benchpark/cmd/unit_test.py

```diff
@@ -204,7 +204,6 @@ def add_back_pytest_args(args, unknown_args):
 
 
 def command(args, unknown_args):
-    global pytest
```

Comment:
@scheibelp New linter version was complaining about these globals https://github.com/LLNL/benchpark/actions/runs/14316951303/job/40125184857, which all 3 of them don't seem necessary, so I removed them instead of adding `# noqa`.  Do you agree?